assets/css/appV2/table.scss

```diff
@@ -12,6 +12,24 @@ th:last-child{
   border-radius: 0 10px 10px 0;
 }
 
+.cardShare::before {
+  content: '';
```

Comment:
I don't know if `.cardShare` is good naming, as it is used for private and shared cards, and only for folder and document cards.
What do you think about something like "folderableCard" ?

Reply:
It's not only for folder and document anymore so maybe .shareableCard or .cardVisibility.

Reply:
I try shareableCard what do you think @AntoineAresu ?

Reply:
Now it's for all personal data you right, what about `.personal-data-card` ?

Reply:
Amazetastic ! I will do it.